### PR TITLE
Transforms: Add sort by transformer

### PR DESCRIPTION
**Pull request by ryantxu**
This transform will sort data before sending it to the visualizations.
image

TODO:
- [x] style for the reverse toggle

### CODE REVIEW

File: public/app/core/components/TransformersUI/SortByTransformerEditor.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                }}
+              />
+            </InlineField>
+            <InlineField
+              label="Reverse"
+              onClick={() => {
+                onSortChange(index, { ...s, desc: !!!s.desc });
+              }}
+            >
+              <Switch
```
Comment (ryantxu) [inline public/app/core/components/TransformersUI/SortByTransformerEditor.tsx:53]: @dprokop is there a way to get this centered reasonably without custom emotion css?

Comment (torkelo) [reply public/app/core/components/TransformersUI/SortByTransformerEditor.tsx:53]: There is an InlineSwitch

**Review by torkelo — APPROVED** [review]
Looks good!

**Comment by osg-grafana** [comment]
@oddlittlebird Do we need to add a label for `add to What’s New`?

**Comment by oddlittlebird** [comment]
Yes, we do. Good catch @osg-grafana.

**Pull request merged**